comp/dogstatsd/replay/def/component.go

```diff
@@ -6,15 +6,12 @@
 // Package server implements a component to run the dogstatsd capture/replay
 //
 //nolint:revive // TODO(AML) Fix revive linter
-package replay
+package def
```

Comment:
Base on the new documentation this package should be name `package replay` https://github.com/DataDog/datadog-agent/pull/25735

cc @dustmop